index.html

```diff
@@ -0,0 +1,611 @@
+<!doctype html>
+<html class="no-js" lang="">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <title>AppBox | Build Beautiful Mobile Apps</title>
+    <meta name="description" content="Best way to build and present your mobile app">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href='https://fonts.googleapis.com/css?family=Roboto:400,400italic,300,500|Raleway:600,500,400,300|Roboto+Condensed:400,700' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/Swiper/3.1.2/css/swiper.min.css">
+    <link rel="stylesheet" href="css/main.css">
+  </head>
+  <body>
+    <header id="top">
+      <h1 class="logo">AppBox</h1>
+      <a id="menuToggle" class="menu-toggle" href="#nav">MENU</a>
+      <nav role="navigation">
+        <ul id="nav" class="menu">
+          <li><a class="nav-link" href="#features">Features</a></li>
+          <li><a class="nav-link" href="#why">Why AppBox</a></li>
+          <li><a class="nav-link" href="#screens">Screenshots</a></li>
+          <li><a class="nav-link" href="#more">More Apps</a></li>
+          <li><a class="nav-link" href="#pricing">Pricing</a></li>
+          <li><a class="nav-link" href="#featured">Featured</a></li>
+          <li><a class="nav-link" href="#support">Support</a></li>
+          <li>
+            <div class="nav-link">
+              <form class="search-field">
+                <label class="" for="search"></label>
+                <input id="search" aria-label="search field" type="search">
+              </form>
+            </div>
+          </li>
+          <li class="close-nav"><a href="#menuToggle"></a></li>
+        </ul>
+      </nav>
+    </header>
+    <main id="content">
+      <section class="hero-section" id="backtotop">
+        <div data-div="best-way">
```

Comment:
This concerns me; why not use a class? Here you're re-implementing inherent semantics. I _might_ allow it if there was a behavioral component (JS functionality tied the `data-` attribute), but to use a `data-` attribute that re-states the current element with a value that could just as easily be a class is not good. 

`data-` attributes are intended to _add_ semantic meaning to an element or to hold additional metadata for scripting purposes. Using a `data-` attribute in this way adds no meaning and tightly couples the styles to the markup. What if someone else comes along and wants to use the `[data-div="best-way"]` styling on an `<article>`?

I see this pattern used throughout this project, so I'm on going to call it out here. Don't do this again.